README.md

```diff
@@ -231,3 +231,55 @@ are some places you can find the games listed in this database:
 - [David Winter's CHIP-8 games](https://www.pong-story.com/chip8/): A collection
   of David Winter's games, which are traditionally included in many emulators
   today.
+
+## Contributing
+
+### Prerequisites
+
+This project uses [NPM][] to manage dependencies and run scripts, including
+[jsonschema](https://www.npmjs.com/package/jsonschema) and
+[Prettier](https://prettier.io/). Follow the instructions to
+[download and install npm on your system](https://nodejs.org/en/download/package-manager).
+
+Once NPM is installed, navigate to the root of this repository and execute
+`npm install` to install this project's dependencies.
+
+### Updating the Database from Octo
+
+The CI system checks to ensure that the database is up to date with the contents
+of the [CHIP-8 Archive], and will fail if there are programs there that are not
+listed in [`programs.json`](./database/programs.json).
+
+Before making your changes, it is advised that you run `npm start` to
+automatically create entries for the new files.
+
+If there are any changes (check with `git status` or `git diff`), review the
+newly generated entries at the end of
+[`programs.json`](./database/programs.json) and perform any cleanup that needs
+to occur. In particular, determine appropriate values for the `"platforms"`
+section for each newly added program.
+
+If you made any changes, run `npm run format` before committing.
```

Comment:
I don't think this is necessary. The Github CI "only" checks to see if everything is formatted correctly, and passes the schema. It does this:

https://github.com/chip-8/chip-8-database/blob/bff63251ff9d0c95b2aab4c4408f92fb4a8d23f9/.github/workflows/test.yaml#L29-L31

Which do:

https://github.com/chip-8/chip-8-database/blob/bff63251ff9d0c95b2aab4c4408f92fb4a8d23f9/package.json#L7-L8

So that's just a Prettier check, and invoking the `validate.js` script, which does this, according to the comment:

https://github.com/chip-8/chip-8-database/blob/bff63251ff9d0c95b2aab4c4408f92fb4a8d23f9/scripts/validate.js#L1-L3

I wouldn't want to burden new contributors with having to update the CHIP-8 Archive before they can add their own programs.

Reply:
I also don't like that being there and would be glad to remove it. In fact I was planning on adding a weekly CI run to create a PR to a la dependabot and removing it then. That PR would need to be a draft for manual review of platforms/etc... but it'd be a nice prompt to action.

The reason I put it there was because I had that issue both times when contributing: because I ran `npm start` to generate hashes. That's the default command I try when I know there are `package.json` scripts to be run.

What do you think about about moving `scripts.start` to `scripts.update`, and not having an `npm start`?

Reply:
Or alternatively having `npm start` run `npm generate-hashes-file && npm format`?

Reply:
That sounds like a great idea 👍